python/ray/ml/tests/test_checkpoints.py

```diff
@@ -4,34 +4,37 @@
 import tempfile
 import unittest
 from typing import Any
-from unittest.mock import patch
 
 import ray
 from ray.ml.checkpoint import Checkpoint
-from ray.ml.tests.utils import mock_s3_sync
+from ray.ml.utils.remote_storage import clear_bucket, get_fs_and_path
 
 
 class CheckpointsConversionTest(unittest.TestCase):
     def setUp(self):
         self.tmpdir = os.path.realpath(tempfile.mkdtemp())
+        self.tmpdir_pa = os.path.realpath(tempfile.mkdtemp())
 
         self.checkpoint_dict_data = {"metric": 5, "step": 4}
         self.checkpoint_dir_data = {"metric": 2, "step": 6}
 
-        self.cloud_uri = "s3://invalid"
-        self.local_mock_cloud_path = os.path.realpath(tempfile.mkdtemp())
-        self.mock_s3 = mock_s3_sync(self.local_mock_cloud_path)
+        # fsspec URI
```

Comment:
Can we emphasize that memory:// and mock:// are all for mocking a cloud target with a convenient local test fs set up?

Reply:
memory:// and mock:// are in-memory filesystems provided by fsspec and pyarrow, respectively. IMO the protocol names are pretty self-explanatory but I'll add a one-line comment to make this clear